src/lxc/cmd/lxc_usernsexec.c

```diff
@@ -375,7 +375,9 @@ int main(int argc, char *argv[])
 
 		ret = unshare(flags);
 		if (ret < 0) {
-			perror("unshare");
+			char *msg = "unshare: CLONE_NEWNS | CLONE_NEWUSER\n"
+				"You may need to enable user namespaces\n";
+			fprintf(stderr, "%s", msg);
```

Comment:
Could you please make this: `"Failed to unshare mount and user namespace\n"`.

Reply:
Done